data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -523,7 +523,7 @@ def _workforce_factor(self) -> bool:
             field_names.LOW_MEDIAN_INCOME_LOW_HS_EDUCATION_FIELD,
         ]
 
-        high_scool_achievement_rate_threshold = (
```

Comment:
Perhaps too particular, but doesn't this break from the state-specific variable assignment for the other factors?

Reply:
I'm not sure I understand the question -- could you say a little more? Thanks!

Reply:
Of course. Like we did here https://github.com/usds/justice40-tool/blob/a06555a4f01cd0eea8b80d7988288b6d6dd9a3bf/data/data-pipeline/data_pipeline/score/score_l.py#L178, or for other criterion, I was curious why we were using in-place assignment of the column here. This seems to be a departure from convention, no?

Reply:
Got it - thanks for clarifying.